src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/LicenceActivationService.kt

```diff
@@ -87,30 +88,17 @@ class LicenceActivationService(
     return releaseDate <= LocalDate.now()
   }
 
-  private fun Licence.isStandardLicenceForActivation(prisoners: List<PrisonerSearchPrisoner>): Boolean {
-    val prisoner = prisoners.firstOrNull {
-      it.prisonerNumber == this.nomsId
-    }
-
-    if (prisoner == null) {
```

Comment:
we now silently filter out llcences that we can no longer find prison records/bookings for.
Otherwise we try to continually process them. 

It seems to only effect a very small number of records - 4 in all time, and trying to handle it at activation doesn't seem to be the right place.